group-generators/generators/hop-protocol-101/index.ts

```diff
@@ -0,0 +1,59 @@
+import { dataProviders } from "@group-generators/helpers/data-providers";
+import { GroupWithData, Tags, ValueType } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+/** GET TEST GROUP BY NAME */
```

Comment:
If you want to test the both groups you can totally do it by returning 2 groups a the end of your Group Generator
Here's an example you can learn from: https://github.com/sismo-core/sismo-hub/blob/main/group-generators/generators/talentlayer/index.ts

Reply:
Thanks for the feedback, changed it accordingly. :)